### PR TITLE
BWS-9: * button styles

### PR DESCRIPTION
**Pull request by Donymak**
image

### CODE REVIEW

File: frontend/src/components/dashboard/components/services-item/services-item.tsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { FC } from 'react';
 import { Button } from 'components/common/common';
 import { ServiceMenuItem } from 'common/types/types';
 import styles from './styles.module.scss';
+import { ButtonStyle } from '../../../../common/enums/ui/button-style.enum';
```
Comment (tbezrukava) [inline frontend/src/components/dashboard/components/services-item/services-item.tsx:5]: #

Comment (tbezrukava) [inline frontend/src/components/dashboard/components/services-item/services-item.tsx]:
```suggestion
import { ButtonStyle } from 'common/enums/enums';
```

File: frontend/src/common/enums/ui/button-style.enum.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 enum ButtonStyle {
   FILLED = 'Filled',
   OUTLINED = 'Outlined',
+  GRAY = 'Gray',
```
Comment (what1s1ove) [inline frontend/src/common/enums/ui/button-style.enum.ts]: this is a color, not style. Create a separate enum `ButtonColor` for it. This enum should have two colors - orange and gray

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: frontend/src/components/common/button/button.module.scss [hunk 3]
```diff
@@ -36,3 +34,20 @@
...
+.btnColorGray {
+  color: var(--color-text-gray);
+  background-color: var(--color-boltgun);
+  border: 2px solid var(--color-boltgun);
+}
+
+.btnColorGray:hover {
+  color: var(--color-text-light);
+  background-color: var(--color-orange);
+  border: 2px solid var(--color-orange);
```
Comment (what1s1ove) [inline frontend/src/components/common/button/button.module.scss]:
```suggestion
  border-color: var(--color-orange);
```

File: frontend/src/components/common/button/button.module.scss [hunk 4]
```diff
@@ -36,3 +34,20 @@
...
+
+.btnColorOrange {
+  color: var(--color-white);
+  border: 2px solid var(--color-orange);
+}
+
+.btnColorGray {
+  color: var(--color-text-gray);
+  background-color: var(--color-boltgun);
+  border: 2px solid var(--color-boltgun);
```
Comment (what1s1ove) [inline frontend/src/components/common/button/button.module.scss]:
```suggestion
  border-color: var(--color-boltgun);
```

File: frontend/src/components/common/button/button.module.scss [hunk 5]
```diff
@@ -36,3 +34,20 @@
 .btnStyleFilled:hover {
   background-color: var(--color-boltgun);
 }
+
+.btnColorOrange {
+  color: var(--color-white);
+  border: 2px solid var(--color-orange);
```
Comment (what1s1ove) [inline frontend/src/components/common/button/button.module.scss]:
```suggestion
  border-color: var(--color-orange);
```

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**